src/display/text_layer.js

```diff
@@ -426,9 +469,67 @@ function renderTextLayer(renderParameters) {
     viewport: renderParameters.viewport,
     textDivs: renderParameters.textDivs,
     textContentItemsStr: renderParameters.textContentItemsStr,
+    textDivProperties: renderParameters.textDivProperties,
+    isOffscreenCanvasSupported: renderParameters.isOffscreenCanvasSupported,
   });
   task._render();
   return task;
 }
 
-export { renderTextLayer, TextLayerRenderTask };
+/**
+ * @param {TextLayerUpdateParameters} renderParameters
+ * @returns {TextLayerRenderTask}
+ */
+function updateTextLayer({
+  container,
+  viewport,
+  textDivs,
+  textDivProperties,
+  isOffscreenCanvasSupported,
+  mustRotate,
+  mustRescale,
```

Comment:
Would it make sense to add defaults as below, since it won't affect the viewer, to ensure that this function works out-of-the-box for third-party users?
```suggestion
  mustRotate = true,
  mustRescale = true,
```